vm/src/frame.rs

```diff
@@ -114,11 +114,16 @@ impl fmt::Debug for Frame {
             .map(|elem| format!("\n  > {:?}", elem))
             .collect::<Vec<_>>()
             .join("");
-        let local_str = "".to_string(); /* self.locals
-            .iter()
-            .map(|elem| format!("\n  {} = {}", elem.0, elem.1.borrow_mut().str()))
-            .collect::<Vec<_>>()
-            .join(""); */
+        let local_str = match self.locals.borrow().kind {
+            PyObjectKind::Scope { ref scope } => match scope.locals.borrow().kind {
+                PyObjectKind::Dict { ref elements } => format!(" {:?}", elements),
```

Comment:
Please use here the `.map` magic like in the value stack to print all locals below eachother.